packages/node_modules/@webex/internal-plugin-conversation/src/encryption-transforms.js

```diff
@@ -192,7 +196,13 @@ export const transforms = toArray('outbound', {
 
   encryptAddActivity: {
     direction: 'outbound',
-    alias: 'encryptVerbActivity'
+    fn(ctx, key, activity) {
+      if (activity.object.objectType === 'reaction2') {
```

Comment:
Let's use the format from the rest of the file for checking like:

```suggestion
      if (has(activity, 'object.objectType') &&  activity.object.objectType === 'reaction2') {
```